test/cli/package-error-missing-import/test.out

```diff
@@ -1,52 +1,44 @@
-foo_class.rb:7: No import provides `Foo::Bar` https://srb.help/5002
+foo_class.rb:7: No import provides `Foo::Bar::OtherPackage::OtherClass` https://srb.help/3718
      7 |      Foo::Bar::OtherPackage::OtherClass # resolves via root
-              ^^^^^^^^
+              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   Autocorrect: Use `-a` to autocorrect
     __package.rb:6: Insert `import Foo::Bar::OtherPackage`
      6 |  # import Foo::Bar::OtherPackage ## MISSING!
                                                      ^
     other/__package.rb:3: Defined here
      3 |class Foo::Bar::OtherPackage < PackageSpec
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-  Note:
-    Try running generate-packages.sh
```

Comment:
Another place where we might want to preserve the gen-packages hint.

Reply:
Restored!